### PR TITLE
content-band columns are 320px wide now

### LINKED ISSUE #177: Post Go Live: Google Search Console
**Google Search Console**
If you have an active Google Search Console with your sitemap uploaded, it may be valuable to get a coverage report and make sure that indexing works as expected. The Google Search Console should be monitored for the weeks after a go-live.

### PR DESCRIPTION
**Pull request by venturewise**
## Issue

Fix #177
## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/newsroom
- After:  https://177-content-band--merative2--hlxsites.hlx.page/newsroom

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -578,6 +578,11 @@ main .columns > div > div > h4:first-child {
     gap: 120px;
   }
 
+  .content-band.columns > div > div {
```
Comment (arumsey) [inline blocks/columns/columns.css:581]: Can you add `.icon-list` here too?

**Comment by arumsey** [comment]
I also had a look https://177-content-band--merative2--hlxsites.hlx.page/events/2023-zelta-user-conference since it uses both `content-band` and `overview` (which adds `picture-circle` to `content-band, icon-list`). Everything looks good there as well.

File: blocks/columns/columns.css (hunk 1 above)
Comment (venturewise) [reply blocks/columns/columns.css:581]: Sounds good, will do.

Comment (arumsey) [reply blocks/columns/columns.css:581]: Can you change this to `.columns.content-band.icon-list > div > div` in order to match with existing `.columns.content-band.icon-list > div` selector that controls the column gap.

**Pull request merged**